recipes/svdb/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "1.0.7" %}
+{% set md5sum = "2764c03866f3aa827bec3fcc50fa1742" %}
+
+package:
+  name: svdb
+  version: "{{ version }}"
+
+source:
+  fn: svdb-{{ version }}.tar.gz
+  url: https://github.com/J35P312/SVDB/archive/SVDB-{{ version }}.tar.gz
+  md5: "{{ md5sum }}"
+
+build:
+  skip: True  # [not py27]
+  number: 0
+
+requirements:
+  build:
+  - cython
+  - python
+  - numpy
+  - scipy
+  - scikit-learn
+  # missing dependency of scipy
+  - libgcc
+
+  run:
+  - python
+  - numpy
+  - scipy
+  - scikit-learn
+  # missing dependency of scipy
+  - libgcc
+
+test:
+  commands:
+  - svdb --help
+
+about:
+  home: https://github.com/J35P312/SVDB
+  license: MIT
```

Comment:
I couldn't find the license declaration in `setup.py` or `README.md` -- are you sure it's MIT licensed?

Reply:
Ah, found the [issue you created](https://github.com/J35P312/SVDB/issues/12), thanks!